docs/src/main/sphinx/installation/try-it.rst

```diff
@@ -0,0 +1,265 @@
+
+Trino runs best on Linux. Our Docker container is built on 
+Linux, which means that you can run this guide from Linux, 
+macOS, or Windows.
+
+-  `Install Docker <https://docs.docker.com/get-docker/>`__
+
+Pull Trino from Docker
+------------------------------
+
+Once Docker is installed, the next step is to pull the Trino container
+from Docker. This container contains both the server and the CLI.
+
+From your terminal, run:
+
+.. code-block:: shell
+
+    docker pull trinodb/trino
+
+Start the Trino server
+----------------------
+
+After you've pulled the container, you'll need to start your Trino server. 
+You're going to make calls against this server (and it's data) throughout 
+this guide. 
+
+.. code-block:: shell
+
+   docker run -p 8080:8080 --name trino trinodb/trino
+
+When the server is ready, you'll see this message:
+
+.. code-block:: text
+
+   INFO    main    io.trino.server.Server    ======== SERVER STARTED ========
+
+Start the Trino CLI
+--------------------
+
+In a new terminal window or tab, run this command to start the Trino
+Interactive CLI:
+
+.. code-block:: shell
+
+    docker exec -it trino trino
+
+When the Trino Interactive CLI is ready to use, you should see:
+
+.. code-block:: shell
+
+    trino>
+
+You can exit the Trino Interactive CLI at any time, run:
+
+.. code-block:: shell
+
+    trino> EXIT;
+
+How to get help
+~~~~~~~~~~~~~~~
+
+One of the most important things you need to know when learning a new
+tool is where to find help. With Trino, it's as easy as passing
+``help;`` to the interactive CLI.
+
+.. code-block:: shell
+
+    trino> help;
+
+Show configured resources
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now that your server is running and you've started the CLI, let's make a
+few calls. We're going to start by listing configured resources
+(``CATALOGS``).
+
+.. code-block:: shell
+
+    trino> SHOW CATALOGS;
+
+The server included with the Docker image will return this response:
+
+.. code-block:: text
+
+      Catalog
+    -----------
+     jmx
+     memory
+     system
+     tpcds
+     tpch
+    (5 rows)
+
+Explore a catalog
+~~~~~~~~~~~~~~~~~
+
+Start with the ``tpch`` catalog, which lets you to test the capabilities
+and query syntax of Trino without configuring access to an external data
+source.
+
+.. code-block:: shell
+
+    trino> SHOW SCHEMAS FROM tpch;
+
+Set catalog and schema
+~~~~~~~~~~~~~~~~~~~~~~
+
+To avoid typing the catalog and schema each time, try the ``USE``
+command:
+
+.. code-block:: shell
+
+    trino> USE tpch.sf100;
+
+Now you should see:
+
+.. code-block:: shell
+
+    trino:sf100>
+
+Show table data
+~~~~~~~~~~~~~~~
+
+To view the tables in sf100, run:
+
+.. code-block:: shell
+
+    trino:sf100> SHOW TABLES;
+
+Which returns:
+
+.. code-block:: text
+
+      Table
+    ----------
+     customer
+     lineitem
+     nation
+     orders
+     part
+     partsupp
+     region
+     supplier
+    (8 rows)
+
+With this list of tables, you can continue to drill-down. Let's look at
```

Comment:
"Too many notes." (Have you seen the movie Amadeus?)
How about:
"Continue to drill down into the `customer` table:
[No hyphen; the verb is drill]